packages/cms-cli/commands/auth.js

```diff
@@ -29,16 +40,25 @@ async function authAction(type, options) {
     process.exit(1);
   }
 
-  if (type !== AUTH_METHODS.oauth.value) {
-    logger.error(
-      `The only supported authentication protocol is '${AUTH_METHODS.oauth.value}'`
-    );
-    return;
-  }
-
-  const configData = await promptUser(OAUTH_FLOW);
   trackCommandUsage(COMMAND_NAME);
-  await authenticateWithOauth(configData);
+  let configData;
+  switch (authType) {
+    case OAUTH_AUTH_METHOD.value:
+      configData = await promptUser(OAUTH_FLOW);
+      await authenticateWithOauth(configData);
+      break;
+    case USER_TOKEN_AUTH_METHOD.value:
+      configData = await userTokenPrompt();
```

Comment:
It looks like this flow assumes that the user is adding a new portal and not modifying an existing entry.

Reply:
Looking at the code in master it seems like that is how it currently works. Do we want to allow this to also be used for updating? I thought that was going to be `hs config`'s job?

Reply:
I think that this must have changed at some point.

Reply:
I'm working on this in a separate PR https://github.com/HubSpot/hubspot-cms-tools/pull/87